### ISSUE TITLE
netsh result parsing bug on some network card

### ISSUE THREAD

**Issue by bennymeg** [ancasicolica/node-wifi-scanner#5]
On some network card on a windows machine, the BSSID block of the netsh result is missing even due the `mode=Bssid` flag is present.

example netsh result will be:
`// SSID 1 : AP-Test1`
`//     Network type              : Infrastructure`
`//     Authentication            : WPA2-Personal`
`//     Encryption                : CCMP`

since you are depending on the mac address of the network card in your parser, it breaks on such situations and an empty object is returned.

**Comment by ancasicolica** [comment]
Could you supply the full output please?

**Comment by bennymeg** [comment]
this is the full output:

Interface name : Wi-Fi
There are 1 networks currently visible.

SSID 1 : myAP
    Network type            : Infrastructure
    Authentication          : WPA2-Personal
    Encryption              : CCMP

I fixed it and submitted a PR..

**Comment by ancasicolica** [comment]
Thanks, I'll check this with another windows machine. I rejected the PR because the tests failed. Didn't look like a solution for me.

**Comment by ancasicolica** [comment]
This is rather strange. I tested it with both windows machines I have access to and both supply more information than your computer does. There is no useful information in the output you supplied, no channel, no RSSI and also no MAC address.

I have two options now:

1. Add networks without additional info: use the SSID (which is optional again) as ID in this case. This would break the current interface
2. Leave it at it is: such networks are ignored as no usable information is supplied by the network adapter


Both "solutions" have their drawbacks. As I have some other issues with Linux (all networks are displayed only as super user) the tool is far from being perfect. So I decided that I'll go with "2", if the windows network card does not support netsh properly, nothing is displayed. Sorry about the inconvenience..
